asteroidsyncserviced/bluez/bluezclient.h

```diff
@@ -24,16 +24,13 @@
 #include <QBluetoothAddress>
 #include <QBluetoothLocalDevice>
 
+#include "../libasteroid/services/common.h"
```

Comment:
This is actually not required in bluezclient.h so it could be included in bluezclient.cpp only :)